src/UI/Forms/Formik/AutosubmitForm.js

```diff
@@ -0,0 +1,26 @@
+import { Form, useFormikContext } from "formik";
+import { isEmpty } from "lodash";
+import { useEffect } from "react";
+
+const AutosubmitForm = ({children, ...otherProps}) => {
+  return <Form {...otherProps}>
+    <SubmitFormOnValuesChange />
```

Comment:
Why do you need this as a component?

Reply:
The only thing I do inside of `SubmitFormOnValuesChange` component is accessing formik context. The context is provided on `Form` level, so we are able to access it from the subtree only (from children of `Form`). 

So, I created this component just to consume the context of the `Form`.